pom.xml

```diff
@@ -295,39 +369,10 @@
             </args>
           </configuration>
         </plugin>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-jar-plugin</artifactId>
-          <version>3.1.0</version>
-          <executions>
-            <execution>
-              <goals>
-                <goal>test-jar</goal>
-              </goals>
-            </execution>
-          </executions>
```

Comment:
Do we know why this execution is no longer needed?

Reply:
The execution produces `test` jar, or jar that contains tests classes. Unless there is another module  or project (dependent) that depends on test classes, producing `test` jar is not necessary.